CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - ADDED: Allows Leaflet version 0.7.7 through 1.x
 
+## [v2.1.0] - 2020-10-01
```

Comment:
How is this file used?

Reply:
It is just documentation. Decided to add this here for consistency. And I thought that GitHub supports this kind of file to show in the interface, but seems like I'm wrong